app/index.js

```diff
@@ -110,6 +111,12 @@ var main = function () {
       next();
     });
   });
+  app.get('/lanip', function(req, res, next) {
+    var iproute = spawn("ip", ["route", "get", "8.8.8.8"], [null, 'pipe', 'ignore']);
```

Comment:
The network frequently fails so the API should return a 404 with details for cases like this:

````sh
$ ip route get 8.8.8.8
RTNETLINK answers: Network is unreachable
```